src/fprime/fpp/visualize.py

```diff
@@ -0,0 +1,136 @@
+""" fprime.fpp.visualize: Command line targets for fprime-viz
+
+@author thomas-bc
+"""
+import argparse
+import subprocess
+from pathlib import Path
+from typing import Callable, Dict, List, Tuple
+
+from fprime.fpp.common import FppUtility
+
+from fprime_visual.flask.app import construct_app
+
+
+def run_fprime_visualize(
+    build: "Build",
+    parsed: argparse.Namespace,
+    _: Dict[str, str],
+    __: Dict[str, str],
+    ___: List[str],
+):
+    """Run pipeline of utilities to generate visualization. This includes:
+    - fpp-to-xml
+    - fpl-convert-xml
+    - fpl-layout
+    - start fprime-visual Flask app to serve visualization
+
+    Args:
+        build: build directory output
+        parsed: parsed input arguments
+        _: unused cmake_args
+        __: unused make_args
+        ___: unused pass-through arguments
+    """
+
+    # We could use the build directory, but it's quirky because not managed by CMake ??
```

Comment:
Thougts?

Reply:
Using `.visualize-cache/` in current directory for now, let me know if you think we should change that.

Reply:
Changed to `/tmp/fprime-visualize`